shell/app-shell/lib/arcs-utils.js

```diff
@@ -14,10 +14,6 @@ import Arcs from './arcs.js';
 const log = Xen.logFactory('ArcsUtils', '#4a148c');
 
 const ArcsUtils = {
-  createArc({id, urlMap, slotComposer, context, loader, storageKey}) {
```

Comment:
I think this is ok, but in general you shouldn't just remove things from arcs-utils because they could be employed by any number of sub-projects that are not visible to you (e.g. I have bunch of server-experiments and other things that remix modules/elements/libs).

Reply:
sorry about that, i'll put it back in - at some point i was tracking what are all the places in the shell that create an arc, but after our chat today i'm no longer concerned about it :)